code/game/objects/effects/gibs.dm

```diff
@@ -18,36 +18,38 @@
 
 /obj/effect/gibspawner/New(location, datum/dna/MobDNA)
 	..()
+	ASSERT(length(gibtypes) == length(gibamounts))
+	ASSERT(length(gibamounts) == length(gibdirections))
 
-	if(istype(loc,/turf)) //basically if a badmin spawns it
+	if(istype(loc, /turf)) //basically if a badmin spawns it
 		Gib(loc, MobDNA)
 
-/obj/effect/gibspawner/proc/Gib(atom/location, datum/dna/MobDNA = null)
-	if(gibtypes.len != gibamounts.len || gibamounts.len != gibdirections.len)
-		to_chat(world, "<span class='warning'>Gib list length mismatch!</span>")
-		return
-
+/obj/effect/gibspawner/proc/Gib(atom/location, datum/dna/MobDNA)
```

Comment:
Proc and variable names should probably be lowercase for consistency with, well, 90% of the codebase.

Reply:
In my defence a most of this was here already, and I made the PR last year.